draft-irtf-cfrg-hpke.md

```diff
@@ -4628,3 +4775,57 @@ L: 32
 exported_value:
 84f3466bd5a03bde6444324e63d7560e7ac790da4e5bbab01e7c4d575728c34a
 ~~~
+
+# Acknowledgements
```

Comment:
It is important that the Acknowledgements come before the test vectors.  They shouldn't be buried after dozens of pages of random-looking data!